shuup/core/utils/forms.py

```diff
@@ -134,17 +137,35 @@ def process_user(self, user_to_recover):
            not user_to_recover.email):
             return False
 
+        uid = urlsafe_base64_encode(force_bytes(user_to_recover.pk))
+        token = self.token_generator.make_token(user_to_recover)
         context = {
             'site_name': getattr(self.request, "shop", _("shop")),
-            'uid': urlsafe_base64_encode(force_bytes(user_to_recover.pk)),
+            'uid': uid,
             'user_to_recover': user_to_recover,
-            'token': self.token_generator.make_token(user_to_recover),
-            'request': self.request,
+            'token': token,
         }
-        subject = loader.render_to_string(self.subject_template_name, context)
-        subject = ''.join(subject.splitlines())  # Email subject *must not* contain newlines
-        body = loader.render_to_string(self.email_template_name, context, request=self.request)
-        email = EmailMessage(from_email=self.from_email, subject=subject, body=body, to=[user_to_recover.email])
-        email.content_subtype = settings.SHUUP_AUTH_EMAIL_CONTENT_SUBTYPE
-        email.send()
+
+        notification_sent = False
+        request = getattr(self, "request", None)
+        if has_installed('shuup.notify') and request and request.shop and self.recover_password_confirm_view_url_name:
+            # Send notification event if any Script enabled
+            from shuup.notify.models import Script
+            from shuup.notify.notify_events import PasswordReset
+            if Script.objects.filter(enabled=True, event_identifier=PasswordReset.identifier).exists():
+                notification_sent = True
```

Comment:
maybe you should just set this if the `PasswordReset` ran without exceptions?

```suggestion
   try:
       PasswordReset(**context).run(shop=self.request.shop)
       notification_sent = True
   except:
        notification_sent = False
```